packages/now-cli/test/integration-v1.js

```diff
@@ -1859,79 +1859,6 @@ test('deploy a static build deployment', async t => {
   t.is(content.trim(), 'hello');
 });
 
-test('use build-env', async t => {
```

Comment:
The `integration-v1.js` file doesn't run in CI. Maybe you meant to remove these tests from the `integration.js` file?

Reply:
Good catch. Updated.